### PR TITLE
Upgrade expfmt vendoring

### PR DESCRIPTION
**Pull request by fabxc**
@beorn7 @juliusv

### CODE REVIEW

File: vendor/vendor.json [hunk 1]
```diff
@@ -173,4 +173,4 @@
 			"revisionTime": "2015-06-24T11:29:02+01:00"
 		}
 	]
-}
+}
```
Comment (beorn7) [inline vendor/vendor.json]: New-line missing.

Comment (juliusv) [inline vendor/vendor.json]: Yeah, `govendor` unfortunately doesn't place newlines at the end of this file :( The only reason why it had a newline before is that I manually edited it for some other reasons. Maybe we should fix that upstream?

**Comment by juliusv** [comment]
:+1: otherwise (whether you keep the newline or not for now)

File: vendor/vendor.json (hunk 1 above)
Comment (fabxc) [inline vendor/vendor.json]: Yes, ugly, but messing with generated files doesn't really solve this either.
:+1: for solving this upstream.

**Comment by fabxc** [comment]
Filed kardianos/govendor#50

**Pull request merged**